### PR TITLE
Minor tweaks to various things

### PR DESCRIPTION
**Pull request by JesterOrNot**
The improvements:

## Pricing

* Make the 'u' in unlimited uppercase as it is at the beginning of that section of text

## About

* Changed usage from 'The headquarters are located in' to 'The headquarters is in'
* Remove unused Quote import

## Education

* As Gitpod is in the cloud it is not only accessible from any device but anywhere.
* Added missing commas
* Fixed spelling of the word 'exercises'
* It should be `Exams & Exercises` not 'Exam & Excercise'
* They are development environments not developer environment

## Recruiting

* Fixed spelling of the word 'exercises'

### CODE REVIEW

File: src/pages/education.tsx [hunk 1]
```diff
@@ -23,14 +23,14 @@ const EducationPage: React.SFC<{}> = () => (
     <IndexLayout 
         canonical='/education/' 
         title="Education" 
-        description="Gitpod simplifies the onboarding process, makes coding accessible from any device and provides a productive learning environment."
+        description="Gitpod simplifies the onboarding process, makes coding accessible from anywhere, and provides a productive learning environment."
```
Comment (nisarhassan12) [inline src/pages/education.tsx:26]: I'm not sure about this change @ChristinFrohne What do you say?

File: src/pages/education.tsx [hunk 2]
```diff
@@ -23,14 +23,14 @@ const EducationPage: React.SFC<{}> = () => (
...
     >
         <div className="grey-container">
 
             <Banner
                 subtitle="Gitpod Education"
                 title={<h1>Let’s <strong>Focus on Teaching Code,</strong><br /> not Tedious Setups!</h1>}
-                paragraph="Gitpod simplifies the onboarding process, makes coding accessible from any device and provides a productive learning environment."
+                paragraph="Gitpod simplifies the onboarding process, makes coding accessible from any device, and provides a productive learning environment."
```
Comment (nisarhassan12) [inline src/pages/education.tsx:33]: @JesterOrNot Thanks. Can you please tell why do we need a comma here? I'm not good at English.

File: src/utils/education.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ export const features = [
         paragraphs: ['Your source code will never be on the developers laptop or workstations, while your new runtimes, libs and frameworks will roll out faster, as they only need to be on the server and not on the developers machines.', 'Self-Hosted Gitpod can run in your corporate network, keep your data on your infrastructure, and does not require and internet connection.']
     },*/
     {
-        title: "Exam & Excercise",
+        title: "Exams & Exercises",
```
Comment (nisarhassan12) [inline src/utils/education.tsx:7]: :+1:

File: src/utils/recruiting.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ export const features = [
     },
     {
         title: "Follow the Progress",
-        paragraphs: ['Track the progress of your candidates in real time as they work on your coding excercises.', 'By using workspace live-sharing, you can interact with candidates during their coding sessions if needed.']
+        paragraphs: ['Track the progress of your candidates in real time as they work on your coding exercises.', 'By using workspace live-sharing, you can interact with candidates during their coding sessions if needed.']
```
Comment (nisarhassan12) [inline src/utils/recruiting.tsx:8]: :+1:

File: src/pages/education.tsx (hunk 2 above)
Comment (JesterOrNot) [reply src/pages/education.tsx:33]: When there are more than two items in a list you add a comma before the and

Comment (nisarhassan12) [reply src/pages/education.tsx:33]: Thanks.

File: src/pages/education.tsx (hunk 1 above)
Comment (ChristinFrohne) [reply src/pages/education.tsx:26]: I'm fine with changing it

**Pull request merged**